MediaManager/Plugin.MediaManager.Abstractions/IVideoPlayer.cs

```diff
@@ -3,5 +3,7 @@ namespace Plugin.MediaManager.Abstractions
 {
     public interface IVideoPlayer : IPlaybackManager
     {
+        IVideoSurface RenderSurface { get; }
```

Comment:
I personally would've chosen another name. `RenderSurface` sounds like a method that does something, not like a property that you could get.